spec/ione/io/io_reactor_spec.rb

```diff
@@ -111,8 +111,8 @@ def fake_connected(connection)
             restarted_future = reactor.start
             crashed = false
             restarted = false
-            stopped_future.on_failure { crashed = true }
-            restarted_future.on_complete { restarted = true }
+            stopped_future.on_rejection { crashed = true }
+            restarted_future.on_fulfillment { restarted = true }
```

Comment:
strictly speaking, the corresponding method is `on_resolution` but I'm questioning if the author intended to implement/call it that way.

Reply:
I've verified that meanwhile. `on_complete` is indeed to be called on both failure and fulfillment so `on_fullfilment` is not the right translation.